x-pack/plugins/security_solution/server/lib/risk_engine/risk_engine_data_client.ts

```diff
@@ -37,10 +62,58 @@ interface RiskEngineDataClientOpts {
   elasticsearchClientPromise: Promise<ElasticsearchClient>;
 }
 
+interface Configuration {
+  enable: boolean;
```

Comment:
```suggestion
  enabled: boolean;
```

"Enabled" is an adjective describing the risk engine; "enable" is a verb that doesn't read as well 👍